content/donate/options/index.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "Donate"
+template = "donate-options.html"
```

Comment:
This feels more like an FAQ: should we rename it?

Reply:
Note that this page also includes the grid of all donation options, so it isn't "just" an FAQ.

Reply:
Does that change your preferences at all?

Reply:
Hmm. A little. I mostly want to make sure that folks who are confused / want to know more have a clear path for where to look.

Reply:
The link to this page from the main donation page feels pretty clear to me:

```html
<div class="donate-faq-link"><a href="/donate/options">Click here</a> for more donation options and FAQs</div>
```